Source/TailwindTraders.Mobile/TailwindTraders.Mobile/Features/Home/HomePage.xaml.cs

```diff
@@ -1,5 +1,7 @@
 ﻿using System.Collections.Generic;
+using System.Diagnostics;
```

Comment:
Is this namespace needed anymore? Is it maybe something you needed while debugging?

Reply:
thx fixed